addOns/retire/src/main/java/org/zaproxy/addon/retire/model/Repo.java

```diff
@@ -307,4 +309,58 @@ private static boolean isGoodCandidate(String version) {
         }
         return isAllZeros != 0; // Not a good value if all zero
     }
+
+    static int getRiskForSeverity(String severity) {
+        if (severity == null || severity.isEmpty()) {
+            severity = "medium";
+        }
+        severity = severity.toLowerCase(Locale.ROOT);
```

Comment:
Could do this normalization when parsing, it would avoid the need to do it each time.